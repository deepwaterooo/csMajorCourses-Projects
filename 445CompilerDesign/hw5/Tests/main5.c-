// C-F13
  ThereIsNoProcedureCalledMainInThisFileToBeFoundAnywhereAtAll()
{
}
