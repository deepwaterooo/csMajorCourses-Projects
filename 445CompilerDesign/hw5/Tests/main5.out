ERROR(LINKER): Procedure main is not defined.
Number of warnings: 0
Number of errors: 1
------------------------------------

Memory Configuration: Data Addresses: 0-9999  Instruction Addresses: 0-9999
Loading file: main5.tm
Halted
Bye.
